templates/telegraf.conf.erb

```diff
@@ -24,12 +24,110 @@
   debug = <%= @debug %>
   quiet = <%= @quiet %>
 
+<% if @outputs -%>
 #
 # OUTPUTS:
 #
-<%= require 'toml-rb'; TomlRB.dump({'outputs'=>@outputs}) %>
+<% require 'toml-rb'
 
+@outputs.sort.each do |key, values|
```

Comment:
quite a lot of changes here. are you able to add some tests to verify the filecontent? We do something similar in the prometheus module:
* https://github.com/voxpupuli/puppet-prometheus/tree/master/spec/fixtures/files
* https://github.com/voxpupuli/puppet-prometheus/blob/master/spec/classes/prometheus_spec.rb#L105-L111

Reply:
This is awesome. The current code only allows to configure ```outputs```, ```inputs```, ```processors``` and ```aggregators```. 

This pull request also allows to disable them. 

@hanej It would be great if you could add the missing unit tests.